options/locale/locale_en-US.ini

```diff
@@ -1411,8 +1411,8 @@ issues.context.edit = Edit
 issues.context.delete = Delete
 issues.no_content = There is no content yet.
 issues.close = Close Issue
-issues.pull_merged_at = `merged commit <a class="ui sha" href="%[1]s"><code>%[2]s</code></a> into <b>%[3]s</b> %[4]s`
-issues.manually_pull_merged_at = `merged commit <a class="ui sha" href="%[1]s"><code>%[2]s</code></a> into <b>%[3]s</b> manually %[4]s`
+issues.pull_merged_at = `merged commit <a class="ui sha" href="%[1]s">%[2]s</a> into <b>%[3]s</b> %[4]s`
+issues.manually_pull_merged_at = `merged commit <a class="ui sha" href="%[1]s">%[2]s</a> into <b>%[3]s</b> manually %[4]s`
```

Comment:
I would replace the HTML with `%s` and use `printf` method like done for `pulls.merged_info_text` to print the HTML into the string. We should aim to eliminate HTML in translations.